packages/airnode-node/src/types.ts

```diff
@@ -287,11 +287,26 @@ export interface WorkerOptions {
   readonly stage: string;
 }
 
-export type WorkerFunctionName = 'initializeProvider' | 'callApi' | 'processProviderRequests';
+export interface InitializeProviderPayload {
+  readonly functionName: 'initializeProvider';
+  readonly state: ProviderState<EVMProviderState>;
+}
+
+export interface ProcessTransactionsPayload {
+  readonly functionName: 'processTransactions';
```

Comment:
One other thing I noticed is the timeout constant is still called `WORKER_PROVIDER_PROCESS_REQUESTS_TIMEOUT` in constants.ts which might be confusing